### ISSUE TITLE
Improve Bot whispers

### ISSUE THREAD

**Issue by MoonPadUSer** [blue-nebula/base#122]
currently bots apparently answer EVERYTIME you whisper somebody, let's instead make it so they don't react to whispers at all but instead give them their own command to interact with.
I propose:
`/bot` and `/bots`
syntax:
```
/bot <bot_name/cn> <command/command_num>
/bots <command/command_num>
```

**Comment by Lagetta** [comment]
I always find it annoying that each time bot would respond to `/whisper` message. This should be fixed. But what if new person won't really understand how to control bots at all? Maybe if someone calls the bot's name, there should be a private interaction that helps newbies to read and how to control them.

Just like now, when you type bot's name, you get the dialogue saying like use this or that to interact with the bot.

**Comment by MoonPadUSer** [comment]
yes, I think providing some kind of tutorial would be kinda helpful
